Files/Interacts/Interaction.cs

```diff
@@ -1148,16 +1148,11 @@ public async void ToggleQuickLook()
         {
             try
             {
-                string selectedItemPath = null;
                 int selectedItemCount;
                 Type sourcePageType = App.CurrentInstance.CurrentPageType;
                 selectedItemCount = (CurrentInstance.ContentPage as BaseLayout).SelectedItems.Count;
-                if (selectedItemCount == 1)
```

Comment:
@duke7553 Will removing this cause any side effects?

Reply:
Not that I can think of.